Iryna_Bondarenko/test/greet_test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+
+var expect = require('chai').expect;
+var greeting = require ('../greet');
+
+
+describe ('the greet function' , function(){
+	it("should return hello Iryna", function() {
```

Comment:
What if someone puts in a name other than Iryna or Maksym?  Your test statement should be less specific, like "should return hello + name"